src/mimeparser.rs

```diff
@@ -274,7 +283,30 @@ impl MimeMessage {
             is_mime_modified: false,
             decoded_data: Vec::new(),
         };
-        parser.parse_mime_recursive(context, &mail, false).await?;
+
+        match partial {
+            None => {
+                parser.parse_mime_recursive(context, &mail, false).await?;
+            }
+            Some(org_bytes) => {
+                let mut text = format!(
+                    "[{} message]",
+                    org_bytes
+                        .file_size(file_size_opts::BINARY)
+                        .unwrap_or_default()
+                );
+                if let Some(delete_server_after) = context.get_config_delete_server_after().await? {
+                    let until = dc_timestamp_to_str(time() + delete_server_after);
+                    text += &*format!(" [Download maximum available until {}]", until);
```

Comment:
Does this `&*` do the same as `.as_str()`?

Reply:
Is this string a replacement for a message? Should it be translatable via stock strings?

Reply:
> Does this `&*` do the same as `.as_str()`?

seems so. `&*` was suggested by my ide, but if you prefer `as_str()` - i do not really care :)

> Is this string a replacement for a message?

yes.

> Should it be translatable via stock strings?

yes. i will move that to stock strings. hard-coded that in the first place as i was not even sure if things will work out finally.